README.md

```diff
@@ -108,16 +108,68 @@ skew may result in `system time has changed` messages in VM logs.
 #### Network
 
 The guest agent uses network interface metadata to manage the network
-interfaces in the guest by performing the following tasks:
-
-*   Enabled all associated network interfaces on boot.
+interfaces by performing the following tasks:
```

Comment:
please add a section mentioning MDS route (i.e. guest agent configurations guarantees that routes to MDS are only created for primary nic).

Reply:
Done.